### ISSUE TITLE
Unable to insert rows if table has many columns

### ISSUE THREAD

**Issue by santolex** [DarkWanderer/ClickHouse.Client#45]
I use `ClickHouseBulkCopy` to insert rows and it throws some http exception. 
In my opinion the problem is that query is too long about 22k symbols long.

**Comment by santolex** [comment]
In [this line](https://github.com/DarkWanderer/ClickHouse.Client/blob/5cfb0f680c58c6517132448a253eaba499483ca2/ClickHouse.Client/Copy/ClickHouseBulkCopy.cs#L151) query string is created and it's too long for http URL in my case.

**Comment by santolex** [comment]
https://github.com/ClickHouse/ClickHouse/pull/2490 might be helpful.

**Comment by DarkWanderer** [comment]
Do you have a reproduction example I can put in a test? (fix is already in, but I'd like to cover it)

**Comment by santolex** [comment]
Here you are.

```cs           
var host = "localhost";
var port = 8123;
var tblName = "default.test_long_insert_tbl";
int columnCount = 600;

var con =new ClickHouseConnection($"Host={host};Port={port}");

//Generating create tbl statement with a lot of columns 
var query = $"CREATE TABLE IF NOT EXISTS {tblName}(\n";
var columns = Enumerable.Range(1, columnCount)
    .Select(x => $" some_loooooooooooooonnnnnnnnnnnngggggggg_column_name_{x} Int32");
query += string.Join(",\n", columns);
query += ")\n ENGINE = MergeTree()\n ORDER BY (some_loooooooooooooonnnnnnnnnnnngggggggg_column_name_1)";

//Create tbl in db
await con.ExecuteStatementAsync(query);

var bulkCopy = new ClickHouseBulkCopy(con) {DestinationTableName = tblName};

var rowToInsert = new[] {Enumerable.Range(1, columnCount).Select(x => (object) x).ToArray()};
await bulkCopy.WriteToServerAsync(rowToInsert);

```

**Comment by DarkWanderer** [comment]
Fix released in `2.2.2.294`

**Comment by DarkWanderer** [comment]
I had to pull `2.2.2.294` from search on NuGet.org temporarily as the new way of handling requests turned out to be not compatible with ClickHouse `20.3`. If you use a recent version of ClickHouse, `2.2.2.294` is still available to install manually (by changing version in .csproj `PackageReference` tag or in `packages.config`)

Reopening this to track fix in next version

(@santolex )

**Comment by DarkWanderer** [comment]
Closing again, as has been properly fixed in `2.3.0`
